client/pages/utils.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Takes Company Data and converts to Trello Style Board.
+ */
+const transformData = arr => {
+  let company_list = {};
+
+  let columns = {
+    applied: {
+      id: "applied",
+      title: "New Applications",
+      companyIds: []
+    },
+    pre_pitch: {
+      id: "pre_pitch",
+      title: "Pre Pitch",
+      companyIds: []
+    },
+    deferred: {
+      id: "deferred",
+      title: "Check In Later",
+      companyIds: []
+    },
+    pitch: {
+      id: "pitch",
+      title: "Pitching",
+      companyIds: []
+    },
+  };
+
+  arr.forEach(elt => {
+    const id = elt['id'];
+    company_list[id] = {
+      id: id,
+      name: elt['name'],
+      description: elt['description']
+    };
+
+    /*
+     * If the company's status fits in one of the columns add it
+     * to that column.
+     */
+    if (elt['status'] in columns) {
+      columns[elt["status"]]['companyIds'].push(id);
+    }
```

Comment:
If it doesn't we should have some kind of error right?

Reply:
Yea good call. I should make the api call pull unarchvied companies as well.